### ISSUE TITLE
Broken by 3.0

### ISSUE THREAD

**Issue by 439bananas** [Iwotastic/Scratch-Boost#7]
The release of Scratch 3.0 has completely broken this in every way I can think of.

**Comment by iwotastic** [comment]
Yeah, Scratch 3.0 has kind of completely broken Scratch Boost. Unfortunately, I have been off Scratch for a while and this summer is the one before my senior year of high school, so I likely will not be able to much time at all to try to rewrite this for 3.0 (despite the fact that it would be really cool). As a result, this repo is now pretty much unmaintained and if it seems right, I may archive it permanently.

If you have any thoughts on this before I archive the repo, that would be pretty cool.

**Comment by 439bananas** [comment]
meh, i kinda liked the concept, it was useful to have at hand

**Comment by Drago-Cuven** [comment]
maybe you can bring this back as a turbowarp extension?

**Comment by Drago-Cuven** [comment]
you wont have to worry about sandboxing anymore

**Comment by qbjl** [comment]
btw i loved this extension back when you could still use it (with nix's s2.0 port) so here's a project that might help: https://scratch.mit.edu/projects/717031217/

this is a 3d extension for scratch that actually works like scratch boost did and adds a category with blocks while NOT being on another instance like turbowarp.
